boards/arm/nrf5340_audio_dk_nrf5340/doc/index.rst

```diff
@@ -0,0 +1,165 @@
+.. _thingy53_nrf5340:
```

Comment:
Not a thingy53 :)

Reply:
Quite picky of you, but OK, I will fix that.